packages/formstr-sdk/src/formstr/formstr.ts

```diff
@@ -163,6 +164,15 @@ export const getFormTemplate = async (formId: string): Promise<V1FormSpec> => {
     if (formVersion === "v0") {
       formTemplate = convertV1Form(formTemplate);
     }
+    if (formTemplate?.metadata?.encryption === "aes") {
```

Comment:
let's move this to a new method? so that people/clients can continue using the previous methods